plugin/login_util.go

```diff
@@ -3,14 +3,15 @@ package gcpauth
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/hashicorp/go-gcp-common/gcputil"
-	"google.golang.org/api/iam/v1"
 	"time"
+
+	"github.com/hashicorp/go-gcp-common/gcputil"
+	"google.golang.org/api/iamcredentials/v1"
 )
 
 func ServiceAccountLoginJwt(
-	iamClient *iam.Service, exp time.Time, aud, project, serviceAccount string) (*iam.SignJwtResponse, error) {
-	accountResource := fmt.Sprintf(gcputil.ServiceAccountTemplate, project, serviceAccount)
+	iamClient *iamcredentials.Service, exp time.Time, aud, serviceAccount string) (*iamcredentials.SignJwtResponse, error) {
```

Comment:
Nit: Can you change the func signature to be a single line?

Reply:
I can, but aside, I would prefer this be several lines I think it's more readable that way... regardless I changed it to a single line in ac07c84dfce173bdb783dab93e8febd37dfac96a as requested